feat(supernets): wrapped tokens for non-ERC-20 to stake (#1723)
maticnetwork/matic-docs: docs/supernets/operate/deploy/staking/stake.md

</details>

> In the following example command, we use the validator key and the rootchain `StakeManager` contract address that were generated earlier. We also set the staking amount to `1000000000000000000` which is equivalent to 1 token. The `--native-root-token` flag is used to specify the address of the native token of the rootchain.

Reviewer: ```suggestion
In the following example command, we use the validator key and the rootchain `StakeManager` contract address that was generated earlier. We also set the staking amount to `1000000000000000000` which is equivalent to 1 token. The `--native-root-token` flag is used to specify the address of the native token of the rootchain.
```

:::info Staking requirement: wrapping a non-ERC-20 token

> Supernets allows for the customization of the gas token and mandates the use of ERC-20 tokens for staking instead of the rootchain's native token. 

Reviewer: ```suggestion
Supernets allow for the customization of the gas token and mandate the use of ERC-20 tokens for staking instead of the rootchain's native token. 
```

> When performing rootchain staking on the Polygon PoS mainnet, [<ins>WMATIC</ins>](https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270?a=0x68b3465833fb72a70ecdf485e0e4c7bd8665fc45) (wrapped MATIC) is the required token. This is due to the ERC-20 standard requirement for staking, which MATIC doesn't meet. 

Reviewer: ```suggestion
When performing rootchain staking on the Polygon PoS Mainnet, [<ins>WMATIC</ins>](https://polygonscan.com/token/0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270?a=0x68b3465833fb72a70ecdf485e0e4c7bd8665fc45) (wrapped MATIC) is the required token. This is due to the ERC-20 standard requirement for staking, which MATIC doesn't meet. 
```

WMATIC is deployed at `0000000000000000000000000000000000001010` as [<ins>MRC20.sol</ins>](https://github.com/maticnetwork/contracts/blob/develop/contracts/child/MRC20.sol) on Bor as one of the genesis contracts. For a detailed explanation, refer to this [<ins>overview</ins>](/docs/pos/bor/core_concepts.md#matic-as-native-token-gas-token).

> This principle also applies to the Mumbai testnet, where wrapped test MATIC must be used instead of test MATIC.

Reviewer: ```suggestion
This principle also applies to the Mumbai Testnet, where wrapped test MATIC must be used instead of test MATIC.
```
